darts/timeseries.py

```diff
@@ -1803,6 +1803,43 @@ def plot(self,
         plt.legend()
         plt.title(self._xa.name);
 
+    def with_columns_renamed(self, cols_name: Union[List[str], str], cols_name_new: Union[List[str], str]) -> 'TimeSeries':
+        """
+        Changes a ts column name and returns a new TimeSeries instance
+        -------
```

Comment:
The docstring doesn't quite match the style we use. It needs a 'Parameters' and a 'Returns' section with appropriate parameter/return value descriptions after the general function description (see other docstrings)